playground/backend/internal/preparators/go_preparators.go

```diff
@@ -51,16 +54,15 @@ func formatCode(args ...interface{}) error {
 	return nil
 }
 
-func changeFileName(args ...interface{}) error {
+func changeGoTestFileName(args ...interface{}) error {
 	filePath := args[0].(string)
-	isUnitTest := args[1].(bool)
-	if isUnitTest {
+	validationResults := args[1].(*sync.Map)
+	isUnitTest, ok := validationResults.Load(validators.UnitTestValidatorName)
+	if ok && isUnitTest.(bool) {
 		testFileName := fmt.Sprintf("%s_test.%s", strings.Split(filePath, sep)[0], goName)
-		cmd := exec.Command(mvCmd, filePath, testFileName)
-		fmt.Println(cmd.String())
-		stdout, err := cmd.CombinedOutput()
+		err := os.Rename(filePath, testFileName)
 		if err != nil {
-			return errors.New(string(stdout))
+			return err
```

Comment:
Maybe add some log here?

Reply:
Why? We return error form preparator and this will be proccesed later